Martium.GuessTheRiddleGame/Program.cs

```diff
@@ -152,6 +151,47 @@ private static void ShowPlayerResult(Dictionary<string, int> playerResult)
                 WriteColoredMessage($" { playerResults.Key} -> {playerResults.Value}", color);
             }
         }
+        private static void StartMainGame()
```

Comment:
maybe just `StartGame` ? (whad do you meant with main ?)

Reply:
well we have method called StartGame before this task when i create while loop for reapeating the game i called it StartMainGame. If you wnat I could put while loop in StartGame method but then it will be huge code in one method i think and that method will be doing multi tasks